config.js

```diff
@@ -2,16 +2,17 @@ var IRCConnector = require("./classes/IRCConnector.class");
 var JSONConnector = require("./classes/persistence/JSONConnector.class");
 
 // Sample Slack-thru-IRC config
-/*module.exports = {
+module.exports = {
```

Comment:
Agreed, the configuration is dependent on the runtime environment anyway, good call!